templates/todo/projects/csharp-sql-swa-func/.repo/bicep/repo.yaml

```diff
@@ -146,8 +146,8 @@ repo:
       to: ./infra
 
     # .devcontainer common (devcontainer.json)
-    - from: ../../../../../common/.devcontainer/devcontainer.json/csharp/func/azure-dev.yml
-      to: ./.devcontainer/azure-dev.yml
+    - from: ../../../../../common/.devcontainer/devcontainer.json/csharp/func/devcontainer.json
```

Comment:
This will protect us if in the future we add `../../../../../common/.devcontainer/devcontainer.json/csharp/func/terraform` 
or any other folder. It's better to be explicit with just the file we want here.